page-data.php

```diff
@@ -29,7 +29,7 @@
   </div>
 
   <aside class="tji-chartview-controls">
-    <div id="js-chartview-controls-toggle" class="tji-chartview-controls__toggle"><i class="fas fa-filter"></i></div>
+    <div id="js-chartview-controls-toggle" class="tji-chartview-controls__toggle"><span>←</span><h4>Chart Filters</h4></div>
```

Comment:
so this is more parser friendly can we use the HTML encoded character here -- `&larr;`
http://left-arrow.net/